### PR TITLE
feat(metrics): pass useragent and client ip to Glean

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

**Comment by chenba** [comment]
@dschom 🥺

**Comment by chenba** [comment]
@vbudhram can you take a look?  (Dan's the one on PTO now 😁 )

File: packages/fxa-auth-server/lib/metrics/glean/server_events.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
+// AUTOGENERATED BY glean_parser v7.2.2.dev8+g91d4c811. DO NOT EDIT. DO NOT COMMIT.
```
Comment (vbudhram) [inline packages/fxa-auth-server/lib/metrics/glean/server_events.ts:5]: Does the old version of Glean parser have issues with passing these values?

**Review by vbudhram — APPROVED** [review]

File: packages/fxa-auth-server/lib/metrics/glean/server_events.ts (hunk 1 above)
Comment (chenba) [reply packages/fxa-auth-server/lib/metrics/glean/server_events.ts:5]: Arkadiusz on the data engineering team generated this file using a not yet released (but PR approved) branch of the glean_parser.

**Pull request merged**